pkg/smartcontract/manifest/abi.go

```diff
@@ -85,14 +86,14 @@ func (a *ABI) IsValid() error {
 			}
 			if methods[i].name == methods[i-1].name &&
 				methods[i].params == methods[i-1].params {
-				return errors.New("duplicate method specifications")
+				return errors.New("ABI: duplicate method specifications")
```

Comment:
Why don't you add `ABI:` outside of `IsValid()`? When you're calling `IsValid()` you know exactly that it's the ABI you're checking and internally it doesn't make a lot of sense, all errors here are ABI-related. Now when you're returning this error to some upper layer, that's where it's important to show that it's the ABI that failed some check.